src/synapse-plugins/converter-plugin/converter-data.vala

```diff
@@ -0,0 +1,214 @@
+/*
+* Copyright (c) 2022 elementary LLC.
+*
+* This program is free software; you can redistribute it and/or
+* modify it under the terms of the GNU General Public
+* License as published by the Free Software Foundation; either
+* version 2 of the License, or (at your option) any later version.
+*
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+* General Public License for more details.
+*
+* You should have received a copy of the GNU General Public
+* License along with this program; if not, write to the
+* Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+* Boston, MA 02110-1301 USA
+*
+* Authored by: Jeremy Wootten <jeremywootten@gmail.com>
+*/
+
+namespace Synapse {
+    const string MASS = "unit of mass";
+    const string LENGTH = "unit of length";
+    const string VOLUME = "unit of volume";
+    const string TIME = "unit of time";
+
+    const SIPrefix [] PREFIXES = {
+        {"yotta", "Y", 1E24},
+        {"zetta", "Z", 1E21},
+        {"exa", "E", 1E18},
+        {"peta", "P", 1E15},
+        {"tera", "T", 1E12},
+        {"giga", "G", 1E9},
+        {"mega", "M", 1E6},
+        {"kilo", "k", 1000},
```

Comment:
```suggestion
        {"kilo", "kg", 1000},
```

See: https://www.wolframalpha.com/input?i=1+kilo

Reply:
I do not think this is right - "kg" is a specific unit of mass whereas "k" is a prefix for "1000" it is also used for "km" (kilo-meter) for example.  I know "kilo" is also commonly used as an abbreviation for "kilogram" but that is not its function here. It might be an idea to add another entry to UNITS to accomodate this abbreviation.

Reply:
I have now added "kilogram" as an entry in UNITS with abbreviation "kilo".

Reply:
> I do not think this is right - "kg" is a specific unit of mass whereas "k" is a prefix for "1000" it is also used for "km" (kilo-meter) for example

You are absolutely right, I got confused.